### DISCUSSION TITLE
How to view documentation for Primevue components included in current Poseidon template (Primevuew 3.26.1)

### DISCUSSION THREAD

**Discussion by dcarapic** [primefaces/community#1932] [PrimeVue Templates] [answered]
Hello,
We've purchased the Poseidon template from you. The template contains Primevuew version 3.26.1. We are having difficulties working with the template because the current official documentation (3.52.0) is in some places very different from version 3.26.1.
For example, the DataTable filtering (https://primevue.org/datatable/#basic_filter) includes components such as IconField which are not available in version 3.26.1.

Is there a way to view the documentation from the Primevue version 3.26.1?

P.S.
Will the templates (in particular the Poseidon template) ever be updated to latest  version of Primevue (3.52.0)? There are some visual artifacts when we simply update the package.json so we are currently 'stuck' on 3.26.1

Thank you.

**Comment by tugcekucukoglu** [comment]
Hello,

Poseidon 3.1.0 uses PrimeVue 3.49.1. Could you try to use it with version 3.52.0?

**Comment by dcarapic** [comment]
Is there an some sort of documentation (or changelog) on how to upgrade from our current Poseidon template to the new template?

**Comment by tugcekucukoglu** [comment, marked as answer]
There are documentation and changelog.

**Reply by dcarapic** [reply]
Wow, you provide a worthless answer after 2 months. Great support :(
